core/src/test/java/org/elasticsearch/action/index/IndexRequestTests.java

```diff
@@ -147,7 +147,7 @@ public void testIndexResponse() {
         assertEquals("IndexResponse[index=" + shardId.getIndexName() + ",type=" + type + ",id="+ id +
                 ",version=" + version + ",result=" + (created ? "created" : "updated") +
                 ",seqNo=" + SequenceNumbersService.UNASSIGNED_SEQ_NO +
-                ",shards={\"_shards\":{\"total\":" + total + ",\"successful\":" + successful + ",\"failed\":0}}]",
```

Comment:
Also just to understand this: this change here only affects the toString() method, not the actual rest response?